web/static/frontend/css/machines-layout.scss

```diff
@@ -64,3 +63,12 @@
     }
   }
 }
+
+.inactive-machine-list {
+  >.machine {
+    display: grid;
+    grid-template-columns: 1fr auto;
```

Comment:
fr units not allowed on `grid-template-columns`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)